sdks/java/core/src/test/java/org/apache/beam/sdk/testing/PAssertTest.java

```diff
@@ -173,7 +173,7 @@ public void testSuccessEncodedDecoded() throws IOException {
    * serializable.
    */
   @Test
-  @Category(ValidatesRunner.class)
+  @Category(NeedsRunner.class)
```

Comment:
I actually think that PAssertTest might be an exception - even though it is a composite transform, it is important to run alongside the VR runner suite to make sure the VR suite is meaningful. At least the particular pipelines that make sure suites are vacuous.

Reply:
Reverted.